mods/cnc/maps/cnc64gdi01/cnc64gdi01.lua

```diff
@@ -0,0 +1,205 @@
+CommandoReinforcements = { "rmbo" }
+MCVReinforcements = { "mcv" }
+
+inf1 = { "e4" }
+auto1 = { "stnk", "stnk" }
+auto2 = { "ftnk", "ftnk" }
+auto3 = { "ltnk", "ltnk", "bike" }
+auto4 = { "arty", "arty", "bike", "bike" }
+auto5 = { "ltnk", "ltnk" }
+auto6 = { "stnk", "stnk" }
+auto7 = { "ltnk", "ltnk" }
+auto8 = { "arty", "arty" }
+
+AutocreateSquads = { auto1, auto2, auto3, auto4, auto5, auto6, auto7, auto8 }
+
+AttackTriggers = { AttackTrigger1, AttackTrigger2, AttackTrigger3, AttackTrigger4 }
+
+harvester = { "harv" }
+
+SamSites = { SAM01, SAM02 }
+Obelisk = { Obelisk01 }
+BiotechCenter = { Biolab }
```

Comment:
You don't need these, you can reference `Obelisk01` and `Biolab` directly (as you already do in some places).